getting-started.md

```diff
@@ -8,4 +8,333 @@ permalink: /getting-started/
+```
+
+Property `activationLevel` defines value of _failed-login-ratio_ on which an activation event will be generated. It will be later used by the _Rule_ to apply defined _Behavior_. In this configuration, the rule will be activated when 65% or more of the logins request will be a failed ones. Property `deactivationLevel` defines value of _failed-login-ratio_ below which deactivation event will be generated disabling protection mechanism. As you can see, we recommend using hysteresis to prevent frequent activation/deactivation but you are free to modify the values to suit your application's characteristics and your own balance between security and user experience. 
+
+Property `window` defines time period for which the ratio will be calculated. Longer periods would consume more memory (unless external data store is used) and would cause slower reaction to changes in traffic patterns.
+
+Property `minimumSampleSize` defines a minimum number of login attempts that needs to occur within `window` for the activation to happen. The reason for this property is that when there is a small number of login attempts, we don't necessarily want to trigger activation. `minimumSampleSize` set to 10 with `window` set to 10 minutes would mean that we are allowing at most 60 failed login attempts (assuming no successful ones) in one hour time. 
+
+Now, we need to define a Rule that would trigger a Behavior on _failed-login-ratio_ activation. Let's again to `WebSecurityConfig` class and add new rule to the builder:
+```java
+    @Bean
+    public FilterConfiguration.BehaviorProviderConfigurer behaviorConfigurer() {
+        return builder -> builder
+                    .rule("usernameLoginOverThreshold")
+                    .when(Conditions::isUsernameLoginOverThreshold)
+                    .then(CAPTCHA)
+                .buildRule()
+                    .rule("failedLoginRatioActive")
+                    .when(Conditions::isFailedLoginRatioActive)
+                    .then(CAPTCHA)
+                .buildRule();
+    }
+```
+We also added _CAPTCHA_ behavior for this rule. 
+
+__And... that's it, time for testing!__
+
+Let's run the application:
+```
+$ ./gradlew bootRun
+```
+
+Navigate to test scenarios and run them:
+```
+$ cd e2e-tests
+$ node test-cs.js
+```
+
+This time the results are following:
+
+{:.table}
+|                      | executed  | failed |
+|----------------------|-----------|--------|
+| `iterations`         | 99        | 0      |
+| `requests`           | 198       | 0      |
+| `test-scripts`       | 396       | 0      |
+| `prerequest-scripts` | 396       | 0      |
+| `assertions`         | 495       | 89     |
+| `total run duration: 16.2s`  
+
+This table is not intuitive to read, but we need to look at __failed__ column in __assertions__ row. We can see 89 login attempts failed because captcha was displayed. If we analyze logs of the application, we would see multiple repetitions of following:
```

Comment:
This explanation about how to interpret the results should rather be placed at the very first mention about the attack simulation.